packages/js/src/ui/components/Notification/DefaultNotification.tsx

```diff
@@ -90,47 +90,49 @@ export const DefaultNotification = (props: DefaultNotificationProps) => {
             'nt-gap-3 nt-shrink-0 nt-ml-6 nt-float-right nt-hidden group-hover:nt-flex'
           )}
         >
-          <Show
-            when={props.notification.isRead}
-            fallback={
+          <Show when={status() !== NotificationStatus.ARCHIVED}>
```

Comment:
the read/unread actions should be only shown for filters: `unread + read, unread`